builtins/util.m4

```diff
@@ -8005,7 +8005,9 @@ define_down_avgs()
 ')
 
 define(`rsqrtd_decl', `
+declare  double @__rsqrt_fast_uniform_double(double)
```

Comment:
Please add the same changes to util-xe.m4

Reply:
It's a good catch! Fixed.